tools/storybook/amp-env/main.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line local/no-module-exports, no-undef
+module.exports = {
+  stories: ['../../../extensions/**/*.*/stories/*.amp.stories.js'],
```

Comment:
Does this have to be `stories/Foo.stories.js`? Could we change that to something that doesn't conflict with Stories?

Reply:
Doesn't have to be, but given that this is "story"book, not sure if we can escape it. Do you have a suggestion for a name?

Reply:
`storybook/foo.js`? I would just emphasize that these are storybooks, not stories.

Reply:
That works for me :))